zip-0000.rst

```diff
@@ -256,13 +299,35 @@ without an explanation.
 Note that it is not the primary responsibility of the ZIP Editors to
 review proposals for security, correctness, or implementability.
 
-Please send all ZIP-related communications either by email to
-<zips@z.cash>, or by opening an issue on the `ZIPs issue
-tracker <https://github.com/zcash/zips/issues>`__. All communications
-should abide by the Zcash Code of Conduct [#conduct]_
+Communicating with the ZIP Editors
+----------------------------------
+
+Please send all ZIP-related communications either:
+- by email to <zips@z.cash>,
+- by opening an issue on the `ZIPs issue tracker <https://github.com/zcash/zips/issues>`, or
+- by sending a message in the `#zips channel on the Zcash R&D Discord <https://discord.com/channels/809218587167293450/809251050741170187>`.
```

Comment:
I added Discord to reflect current practice. Should we add an invite link as well?